.travis/osx..script.sh

```diff
@@ -12,3 +12,6 @@ cmake $CMAKE_FLAGS -DUSE_WERROR=OFF ..
 make -j4
 make tests
 ./tests/tests
+
+# TODO
```

Comment:
We use `make dmg`. Note, the prefix path must be configured or else bad things will happen.  This is all documented here: https://github.com/LMMS/lmms/wiki/Compiling#macos-packaging